test/autotest.py

```diff
@@ -525,7 +527,9 @@ def testRestart():
       if i.endswith(".dmtcp"):
         cmd+= " "+ckptDir+"/"+i
     #run restart and test if it worked
-    procs.append(launch(cmd))
+    procs.append(runCmd(cmd))
+    #wait and see if process will die shortly after restart
+    sleep(S*SLOW)
```

Comment:
Same story here. This `sleep` should go _after_ the following `WAITFOR` statement.